fvm/fvm_blockcontext_test.go

```diff
@@ -1671,10 +1672,13 @@ func TestBlockContext_UnsafeRandom(t *testing.T) {
 	chain, vm := createChainAndVm(flow.Mainnet)
 
 	header := &flow.Header{Height: 42}
+	snapshot := pmock.Snapshot{}
+	snapshot.On("RandomSource").Return(unittest.RandomBytes(48), nil)
```

Comment:
you could also use the fixture here: https://github.com/onflow/flow-go/blob/8d6767eaf7b0a88f1f97ce0f8217f7bed7a65a28/engine/execution/testutil/fixtures.go#L635

Reply:
I reorganized the code and added a new function `ProtocolSnapshotFixture` to be used when only a snapshot is needed